File: web/fhx/fhx.service.go
package fhx

import (
	"fmt"
	"net/http"

	"github.com/julienschmidt/httprouter"
	"github.com/schlucht/deltadoku/web/cors"
)

func SetupRouter(router *httprouter.Router) {
	router.POST("/api/fhx/:fhxText", cors.Cors(postReadFhx))
}

func postReadFhx(w http.ResponseWriter, r *http.Request, fhx httprouter.Params) {

	data := fhx.ByName("fhxText")

	fmt.Printf("%v\n", data)

	// w.Header().Set("Access-Control-Allow-Origin", "*")
	w.Header().Set("Content-Type", "text/plain")
	w.WriteHeader(http.StatusAccepted)
	w.Write([]byte(data))

}

File: frontend/src/services/fhxService.js

import http from "./http-commons";

const getUpData = async (fhxText) => {
        try {
            const response =  await http.post('/fhx/fhxText=' + fhxText)
            const fhx = await response.data
            return fhx

        } catch( error ) {
            const err = new Error("http axio Error (getUpData): " + error)
            throw(err)
        }

    }

export { getUpData }